census/views.py

```diff
@@ -160,7 +182,7 @@ def fetch_events_from_db(self, **kwargs):
         search = kwargs.get('search')
         user_auth_status = kwargs.get('user_auth_status')
 
-        if user_auth_status:
+        if not user_auth_status:
```

Comment:
I accidentally got this logic the other way around in my last PR. Correcting it here.